modules/nf-core/poolsnp/main.nf

```diff
@@ -0,0 +1,71 @@
+process POOLSNP {
+    tag "$meta.id"
+    label 'process_medium'
+    // WARN: Version information not provided by tool on CLI. Please update version string below when bumping container versions.
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/poolsnp:1.0.1--py312h7e72e81_0':
+        'biocontainers/poolsnp:1.0.1--py312h7e72e81_0' }"
+
+    input:
+    tuple val(meta)      , path(mpileup)
+    tuple val(meta2)     , path(reference)
+    val(max_cov)
```

Comment:
I think this is the only required parameter with no default, so you don't need to expose the others as value channels.

Reply:
Removed the parameters with default values, also removed the corresponding entries from `meta.yml`, these parameters shouldn't be included in the `meta.yml` right?

Passing the other parameters through `ext.args` as we get an empty `.vcf` file with the default values.

Reply:
Ok, definitely sensible to pass the other parameters for the test to make an actual output